plugins/ObjectsFormat.cc

```diff
@@ -261,15 +261,15 @@ void ObjectsFormat::FillJetType(JetType& I, const pat::Jet* R, bool isMC) {
     I.chm         = R->chargedHadronMultiplicity();
     I.npr         = R->chargedMultiplicity() + R->neutralMultiplicity();
     I.flavour     = R->partonFlavour();
-    if(isMC && R->genParton()) I.mother = false;//Utilities::FindMotherId(dynamic_cast<const reco::Candidate*>(R->genParton()));
+    if(isMC && R->genParton()) I.mother = Utilities::FindMotherId(R->genParton());
+    I.isMatched   = (I.mother==25);
     I.isLoose     = R->hasUserInt("isLoose") ? R->userInt("isLoose") : false;
     I.isMedium    = false;
     I.isTight     = R->hasUserInt("isTight") ? R->userInt("isTight") : false;
     I.isTightLepVeto     = R->hasUserInt("isTightLepVeto") ? R->userInt("isTightLepVeto") : false;
-    I.isCSVL      = false;//IsBTagged(Tagger, 1, I.CSV);
-    I.isCSVM      = false;//IsBTagged(Tagger, 2, I.CSV);
-    I.isCSVT      = false;//IsBTagged(Tagger, 3, I.CSV);
-    I.isMatched   = (I.mother==25);
+    I.isCSVL      = R->bDiscriminator("pfCombinedInclusiveSecondaryVertexV2BJetTags")>0.460 ? true : false;
+    I.isCSVM      = R->bDiscriminator("pfCombinedInclusiveSecondaryVertexV2BJetTags")>0.800 ? true : false;
+    I.isCSVT      = R->bDiscriminator("pfCombinedInclusiveSecondaryVertexV2BJetTags")>0.935 ? true : false;
 }
```

Comment:
In the VZ(lep) analysis we actually don't use b-tagging at all. Lisa is starting to have a look at the b-tagging for the 0-leptons VZ analysis but so far I think she's still used the CSV discriminator out of the box with no SF applied.